docs/examples/language_model/language_model.ipynb

```diff
@@ -24,13 +33,17 @@
    "source": [
     "## Language model definition - one sentence\n",
     "\n",
-    "A Language model tries to predict the next word given the previous ones.\n",
+    "The standard approach to language modeling consists of training a model that given a trailing window of text, predicts  the next word in the sequence. When we train the model we feed in the inputs $x1, x_2, ...$ and try at each time step to predict the corresponding next word $x_2, ..., x_{n+1}$. To generate text from a language model, we can iteratively predict the next word, and then feed this word as the input to the model at the subsequent time step.\n",
+    "\n",
+    "<div style=\"width: 500px;\">![Language Model](https://gluon.mxnet.io/_images/recurrent-lm.png)</div>\n",
```

Comment:
It may be better to directly use proper HTML:
`<img src="https://gluon.mxnet.io/_images/recurrent-lm.png" style="width: 500px;"/>`

Reply:
I adopted this from Zack's notebook. Agreed, I can update.

Reply:
Ok, thanks.